javagen/src/main/java/com/azure/autorest/template/ModelTemplate.java

```diff
@@ -1166,7 +1167,9 @@ private static boolean isGenerateConstantEmptyByteArray(ClientModel model, JavaS
      */
     private static boolean modelRequireSerialization(ClientModel model) {
         // TODO (weidxu): any other case? "binary"?
-        return !ClientModelUtil.isMultipartModel(model);
+        return !ClientModelUtil.isMultipartModel(model)
+                // not GroupSchema
+                && !(model.getImplementationDetails() != null && model.getImplementationDetails().getUsages() != null && model.getImplementationDetails().getUsages().contains(ImplementationDetails.Usage.OPTIONS_GROUP));
```

Comment:
currently, 2 condition for a model class to have no serizliation
1. multipart/form-data (and related `##FileDetails` model), as the serialization is via a Helper class.
2. this GroupSchema (here we call it OPTIONS_GROUP, welcome suggestion on naming)